ports/atmel-samd/mpconfigport.h

```diff
@@ -73,9 +73,9 @@
 #define MICROPY_PY_SYS_PLATFORM                     "MicroChip SAMD51"
 #elif defined(SAME54)
 #define MICROPY_PY_SYS_PLATFORM                     "MicroChip SAME54"
-#ifndef MICROCONTROLLER_VOLTAGE_DISABLE
-#define MICROCONTROLLER_VOLTAGE_DISABLE (1)
-#endif
+// #ifndef MICROCONTROLLER_VOLTAGE_DISABLE
+// #define MICROCONTROLLER_VOLTAGE_DISABLE (1)
+// #endif
```

Comment:
Please delete this instead of commenting it out.  In fact, all traces of MICROCONTROLLER_VOLTAGE_DISABLE can be removed as there's no longer any use of it.

[this is the bit that I intended to write along with my review]

Reply:
Removed all references to MICROCONTROLLER_VOLTAGE_DISABLE